internal/ignition/installmanifests.go

```diff
@@ -978,6 +986,18 @@ func (g *installerGenerator) updateIgnitions() error {
 		}
 	}
 
+	// An ignition for arbiter nodes will only be generated for clusters that include arbiter nodes
+	isArbiterExist := common.IsClusterTopologyHighlyAvailableArbiter(g.cluster)
```

Comment:
nit, but maybe call this `arbiterExists` or `isArbiterCluster`. I think it just reads a bit better